cd2024bfs2g1-frontend/src/main/ngx/src/app/main/my-packs/pack-client-detail/pack-client-detail.component.ts

```diff
@@ -163,12 +163,12 @@ export class PackClientDetailComponent implements OnInit{
 
   getDays() {
     const filter = {
-      pck_id: this.route.snapshot.params["pck_id"],
+      pck_id: this.pckId,
```

Comment:
aqui probade a usar `this.pckId.toString()` e xa non vos deberia facer falta usar outro endpoint